### DISCUSSION TITLE
Switch between Scratch 2.0 (or Scratch 3.0) designs in editor

### DISCUSSION THREAD

**Discussion by gometrysten** [ScratchAddons/ScratchAddons#1762] [Ideas]
The title describes it all. It may be hard to do because you will need to find all the textures etc.
But [here](https://nathan.github.io/pixie/)'s something I found (sorta glitchie).

**Comment by just-a-github-user** [comment]
Bump

**Reply by retronbv** [reply]
There is no need to bump.

**Comment by Secret-chest** [comment]
> The title describes it all. It may be hard to do because you will need to find all the textures etc.
> But here's something I found (sorta glitchie).

That see inside animation...

**Reply by retronbv** [reply]
I agree it looks amazing

**Comment by CopperCopleer** [comment]
WOW AMAZING!

**Comment by mxmou** [comment]
I'm working on an addon that will make the editor look like 1.4 or 2.0.

**Comment by CopperCopleer** [comment]
oh boy! 1.4!

**Comment by Secret-chest** [comment]
The semi-dark 1.4 theme is very easy for the eyes
